packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -417,7 +390,7 @@ export const GRID_COLUMN_ORDER_CHANGE = 'columnOrderChange';
  * @ignore - do not document.
  * @event
  */
-export const GRID_ROWS_UPDATED = 'rowsUpdated';
+export const GRID_ROWS_UPDATE = 'rowsUpdate';
```

Comment:
GRID_ROWS_CHANGE then? 
as you remove 'update'  in columns

Reply:
@dtassone and how would you name the two other events? `GRID_ROWS_SET`, `GRID_ROWS_CLEAR`.

If we update the row update events to only fire one, then `GRID_ROWS_CHANGE` seem definitely better.

Reply:
What's the point to have `GRID_ROWS_CLEAR`? It's only used by the sorting to clear the sorting once all the records are updated. `GRID_ROWS_SET` could be used instead.

Reply:
Well I like granular events and those are for internal use but ultimately they are change events.
However the clear and set occurs in 1 rendering so you can't publish change as it's for internal reset of rows to put new ones. Users should only know about the new updated set.

Reply:
If we have doubts, how about we keep those for the next iteration?

Reply:
Looking at the publish:

- When GRID_ROWS_CLEARED is triggered, GRID_ROWS_SET is always triggered.

At the listeners:

- GRID_ROWS_SET is listed by `applySorting`, `applySorting`
- GRID_ROWS_CLEARED is listened by "empty sorting"
- GRID_ROWS_UPDATED is listened by `applySorting`, `applyFilters`

So it seems that we only need one event.

Reply:
GRID_ROWS_CLEARED allows to clear the sortedRows state.

Reply:
the split between GRID_ROWS_UPDATED and GRID_ROWS_SET was more for a user perspective where one would want to subscribe to row updates

Reply:
`GRID_ROWS_SET` is published by `setRows`. But this method is mostly used in demos that programmatically call the API, so the user that calls the method doesn't need an event and we can remove it. The only use of `setRows` outside demos is in `updateRows`, which also publishes `GRID_ROWS_UPDATED `.

Side note: `setRows` is actually appending rows, renaming to `addRows` might be better.

Reply:
Originally the idea was to do a reset with a clear and a set, to avoid sortedRows error... If we remove the clear, we can remove the set as well and just rename the updated to change.